framework/auth/oauth_scopes.py

```diff
@@ -124,7 +124,7 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ, CoreScopes.ALWAYS_PUBLIC, )
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
     USERS_CREATE = USERS_READ + (CoreScopes.USERS_CREATE, )
-    USERS_EMAIL = (CoreScopes.USERS_EMAIL, )
+    USER_EMAIL_READ =  (CoreScopes.USERS_READ, CoreScopes.USER_EMAIL_READ, )
```

Comment:
@cwisecarver, seems like it. The comments around `ALWAYS_PUBLIC` seem a bit conflicting to me though. Every scope that CAS recognizes ends up getting `ALWAYS_PUBLIC`?

```
# NOTE: Use with extreme caution.
# This should NEVER be assigned to endpoints:
#    - with mutable data,
#    - that might contain *anything* that could be personally-identifiable,
#    - as a write scope
ALWAYS_PUBLIC = 'always_public'
```

```
class scope(namedtuple('scope', ['parts_', 'description', 'is_public'])):
    """ Patch to add `ALWAYS_PUBLIC` scope to every selectable scope,
        ensuring that public endpoints are accessible with any token.
    """
    @property
    def parts(self):
        return frozenset((CoreScopes.ALWAYS_PUBLIC, )).union(self.parts_)
```